frontend/src/lib/components/ServiceRequestDetails.svelte

```diff
@@ -18,7 +18,7 @@
 		visible = !visible;
 	}
 
-	function getStatus (serviceRequest: ServiceRequest) : string  {
+	function getStatus(serviceRequest: ServiceRequest): string {
```

Comment:
If you remove the `: string` as the return type, the type error will go away

Reply:
That will allow typescript to infer the return type which will mean it will be `"success" | "warn" | "error"`